### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by mugberto**
# Project: Bookstore CMS
In the branch, we added style to our app to match the UI design given [here](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)

To log in to Zeplin, use the following credentials:
- username: microverse_student
- password: c9A6EEh8hTam

### CODE REVIEW

File: src/components/appComponents/booksComponents/AddBookForm.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: #c4c4c4;
+}
+
+.cat-input {
+  width: 17.813rem;
```
Comment (Davidosky007) [inline src/components/appComponents/booksComponents/AddBookForm.css]: * Kindly adjust the width to fit the screen currently the width overflows causing a scroll bar to appear below 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @mugberto 👋, 

Good job so far! 💪 
* The repo looks professional.✔️
* Git/GitHub flow best practices have been followed✔️
* There are no linter errors✔️
* You have descriptive PR✔️
## * There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by mugberto** [comment]
Hi @Davidosky007, I added a little bit of responsiveness to solve the issue. Before, I had used the exact measures from the design as requested.

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @mugberto

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**